src/generateNumberCards.js

```diff
@@ -0,0 +1,25 @@
+function randomInt(max, min) {
+    return Math.floor(Math.random() * (max - min + 1) + min)
+}
+
+function createMultipleCardsBetween3And5(){
```

Comment:
Now that you have the functionality working, maybe change this function to createCards or something simple?